src/Worker.Extensions.DurableTask/DurableTaskExtensionStartup.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Core;
+using Microsoft.Azure.Functions.Worker.Extensions.DurableTask;
+using Microsoft.Extensions.Hosting;
+
+[assembly: WorkerExtensionStartup(typeof(DurableTaskExtensionStartup))]
```

Comment:
This line triggers the .NET worker to automatically run our `Configure` method below. This is how we inject our middleware without any customer action (besides referencing our extension).